YMChat/src/main/java/com/yellowmessenger/ymchat/YMChat.java

```diff
@@ -108,6 +117,40 @@ private boolean validate(Context context) throws Exception {
         return true;
     }
 
+    private boolean isImageUrl(String customLoaderUrl) {
+        if(isValid(customLoaderUrl)){
+            String regex = "([^\\s]+(\\.(?i)(jpe?g|png|gif|svg))$)";
```

Comment:
Please check if this is working https://avatars.githubusercontent.com/u/7659

Reply:
@kaunteya  It will not, it should end with above mentioned extensions

Reply:
As discussed, removed extension check